oe-account-service/op-energy-account-service/src/OpEnergy/Account/Server/V1/DB.hs

```diff
@@ -44,6 +44,11 @@ import           Data.OpEnergy.Account.API.V1.BlockTimeStrikeGuess
 import           Data.OpEnergy.API.V1.Positive(fromPositive)
 import           Data.OpEnergy.API.V1.Natural(verifyNatural, fromNatural)
 import           OpEnergy.Account.Server.V1.DB.Migrations
+import           OpEnergy.BlockTimeStrike.Server.V1.DB.Migrations.SplitBlockTimeStrikeObservedFromBlockTimeStrike
+                   ( transformBlockTimeStrikeGuessGuessFromTextToBool
+                   , splitBlockTimeStrikeObservedFromBlockTimeStrike
+                   , createBlockTimeStrikeObservedTable
+                   )
```

Comment:
changes responsible for migration  which splits BlockTimeStrikeObserved from BlockTimeStrike: